src/config/config.js

```diff
@@ -1,7 +1,7 @@
 const Config = {
-  AUTH0_DOMAIN: process.env.REACT_APP_AUTH0_DOMAIN,
-  AUTH0_CLIENT_ID: process.env.REACT_APP_AUTH0_CLIENT_ID,
-  AUTH0_REDIRECT_URI: process.env.REACT_APP_REDIRECT_URI,
+  AUTH0_DOMAIN: 'dev-8buiztfssus4palq.us.auth0.com',
+  AUTH0_CLIENT_ID: 'LyOGzbSP5BjkqWQiRuCqxZz1em74jvaM',
```

Comment:
Here we need not add these values like client ID and domain, it will fetch from pipeline automatically, and if we want to run locally we can manually add it and we should not commit it.

Reply:
These are for local setup ,got committed by mistake will revert it.